plugins/continuous-toolbox/src/ContinuousFlyout.js

```diff
@@ -72,6 +86,42 @@ export class ContinuousFlyout extends Blockly.VerticalFlyout {
     return null;
   }
 
+  /**
+   * Selects an item in the toolbox based on the scroll position of the flyout.
+   * @param {number} position Current scroll position of the workspace.
+   */
+  selectCategoryByScrollPosition(position) {
```

Comment:
Does this need to be public?

Reply:
Made everything either private (with underscores) or package as needed. Thanks!